modules/fedimint-ln-common/src/lib.rs

```diff
@@ -202,6 +202,12 @@ pub struct LightningGateway {
     pub gateway_id: secp256k1::PublicKey,
 }
 
+impl LightningGateway {
+    pub fn is_expired(&self) -> bool {
+        self.valid_until < fedimint_core::time::now()
```

Comment:
Not an issue for this PR, but until this https://github.com/fedimint/fedimint/issues/2573 is solved, this function could potentially return different results if called on the server or client side

Reply:
I have a draft PR out for that (#3289)